packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -241,15 +241,6 @@ Future<DateTime?> showDatePicker({
       locale: locale,
       child: dialog,
     );
-  } else {
-    final DatePickerThemeData datePickerTheme = DatePickerTheme.of(context);
-    if (datePickerTheme.locale != null) {
-      dialog = Localizations.override(
-        context: context,
-        locale: datePickerTheme.locale,
-        child: dialog,
-      );
-    }
```

Comment:
Why remove this block of code?

Reply:
Oops, good catch, it was to allow compiling in my project that is not on the dev flutter branch. I'll revert it